### ISSUE TITLE
[FEATURE] 📜 Custom Plugins Log Levels

### ISSUE THREAD

**Issue by jodydonetti** [ZiggyCreatures/FusionCache#128]
### Is your feature request related to a problem? Please describe
Currently FusionCache allows customization of a lot of different log levels to be used in different scenarios, such as for serialization errors, factory errors, factory synthetic timeouts, backplane errors, etc.

When talking about plugins though, this is not the case: every log message is either `Information` (for things like "a plugin has been added") or `Error` (for things like when an error occurred while adding a plugin, etc).

### Describe the solution you'd like
Based on [a request](https://github.com/ZiggyCreatures/FusionCache/discussions/122) by community member @shoaibshakeel381 , FusionCache may add a couple of custom log levels, related to plugins.

In particular, 2 new options will be added in the `FusionCacheOptions` class:

```csharp
public LogLevel PluginsInfoLogLevel { get; set; }
public LogLevel PluginsErrorsLogLevel { get; set; }
```

Their default values will be, respectively, `LogLevel.Information` and `LogLevel.Error`.

**Comment by jodydonetti** [comment]
Hi all, I just release [v0.20.0-preview2](https://github.com/ZiggyCreatures/FusionCache/releases/tag/v0.20.0-preview2) 🎉

Unless some problems will came up this week, the next weekend I'll release the final **v0.20.0**.

**Comment by jodydonetti** [comment]
Hi everyone, I've finally released the new [v0.20.0](https://github.com/ZiggyCreatures/FusionCache/releases/tag/v0.20.0) 🎉
